models/websocket_models.go

```diff
@@ -31,7 +29,7 @@ const (
 	CodeAttack
 	CodeEndGame
 	CodeInvalidSignal
-	CodeSignalAbsent  // if the req msg does not contain "code" field
+	CodeSignalAbsent // if the req msg does not contain "code" field
```

Comment:
Why do we need this?

Reply:
Any payload that doesn't have `Code` in its JSON is invalid and we catch that edge case with that

Reply:
yeah I got that, it's just that this is not the right PR for this change. Not blocking though just wanted to flag that for future PRs